OpenOversight/test_data.py

```diff
@@ -21,22 +21,32 @@ def populate():
     po4 = models.Officer(last_name='Cuddleme', first_name='Haywood', middle_initial='U',
                          race='HISPANIC', gender='F', birth_year=1950, pd_id=1,
                          employment_date=datetime(2014, 4, 4, 1, 1, 1))
+    po5 = models.Officer(last_name='Klozoff', first_name='Oliver', middle_initial='U',
+                         race='WHITE', gender='M', birth_year=1950, pd_id=1,
+                         employment_date=datetime(2004, 4, 4, 1, 1, 1))
+    po6 = models.Officer(last_name='O\'Problem', first_name='Bea', middle_initial='U',
```

Comment:
these changes are making it increasingly clear we need to just iterate over something to create test data, which I'll try to grab this week.